SPECS-EXTENDED/osinfo-db/osinfo-db.spec

```diff
@@ -1,24 +1,62 @@
-Summary:        osinfo database files
-Name:           osinfo-db
-Version:        20221130
-Release:        2%{?dist}
-License:        LGPL-2.1-or-later
-Vendor:         Microsoft Corporation
+%define with_mingw 0
+%if 0%{?fedora}
+    %define with_mingw 0%{!?_without_mingw:1}
+%endif
```

Comment:
We will not be using mingw and fedora macros. Can you please clean up the spec file.